src/main/java/net/dv8tion/jda/api/utils/messages/MessageCreateData.java

```diff
@@ -253,6 +253,16 @@ public boolean isTTS()
         return tts;
     }
 
+    /**
+     * Whether this message is silent.
+     *
+     * @return True, if the message will not trigger push and desktop notifications
+     */
+    public boolean isSilent()
+    {
+        return flags == 1 << 12;
```

Comment:
Check `isSuppressEmbeds` for how to implement this.

Reply:
Done.